### PR TITLE
Prepare for Release v11.0

### PR DESCRIPTION
**Pull request by asalkeld**
```release-note
NONE
```

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+## v11.0
+
+- Add PrivateAPIServer boolean to the new public API ([#2001](https://github.com/openshift/openshift-azure/pull/2001), [@asalkeld](https://github.com/asalkeld), 01/11/2019)
+- Remove  PeerVnetID from 2019-10-27-preview API ([#2044](https://github.com/openshift/openshift-azure/pull/2044), [@asalkeld](https://github.com/asalkeld), 31/10/2019)
+- Change the API for Private Cluster as per new agreed model ([#2026](https://github.com/openshift/openshift-azure/pull/2026), [@asalkeld](https://github.com/asalkeld), 25/10/2019)
```
Comment (ehashman) [inline CHANGELOG.md]: Huh, no changenote on the 3.11.153 update? :woman_shrugging:

**Comment by asalkeld** [comment]
flake #1515
/retest

**Comment by ehashman** [comment]
Couple of flakes: #1515 

/retest

**Comment by asalkeld** [comment]
flake #1515
/retest

**Comment by jim-minter** [comment]
/retest

@ehashman @asalkeld has `osarpint.azurecr.io/openshift-on-azure/azure:v11.0` been pushed or is that being done after this merges?  I can't pull the image but not sure if that's me or not.

**Comment by ehashman** [comment]
I haven't pushed anything, I think Angus is going to. We don't have the v11.0 upgrade job set up yet so it's not being tested.

**Comment by asalkeld** [comment]
/retest
try once more..

**Pull request merged**